src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenOnBehalfOfParameterBuilder.cs

```diff
@@ -85,6 +87,27 @@ public AcquireTokenOnBehalfOfParameterBuilder WithForceRefresh(bool forceRefresh
             return this;
         }
 
+        /// <summary>
+        /// To help with resiliency, AAD Cached Credential Service (CCS) operates as an AAD backup.
+        /// </summary>
+        /// <param name="userObjectIdentifier">GUID which is unique to the user, parsed from the client_info.</param>
+        /// <param name="tenantIdentifier">GUID format of the tenant ID, parsed from the client_info.</param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public AcquireTokenOnBehalfOfParameterBuilder WithCcsRoutingHint(string userObjectIdentifier, string tenantIdentifier)
```

Comment:
This code is same as [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2713/files#diff-5197c1f0bc80f2a410918064894a2a01956c5d6426f84d680ff1492a2204f508R112). Can this be reused?